### DISCUSSION TITLE
github.com/google/gopacket/pcap go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap_unix.go:34:10: fatal error: pcap.h: No such file or directory    34 | #include <pcap.h>       |          ^~~~~~~~

### DISCUSSION THREAD

**Discussion by djallalzoldik** [projectdiscovery/naabu#744] [General]
github.com/google/gopacket/pcap
go/pkg/mod/github.com/google/gopacket@v1.1.19/pcap/pcap_unix.go:34:10: fatal error: pcap.h: No such file or directory
   34 | #include <pcap.h>
      |          ^~~~~~~~

**Comment by Mzack9999** [comment]
@djallalzoldik You need to install libpcap-dev on your operative system:
```
# ubuntu
$ apt install libpcap-dev
# mac
$ brew install libpcap

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
